cloud: update release notes (#10264)
pingcap/docs: tidb-cloud/release-notes-2022.md

This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.

## August 30, 2022

**General changes**

* Support AWS PrivateLink-powered endpoint connection as a new network access management option for TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.

>   The endpoint connection is secure and private, and does not expose your data to the public internet. In addition, the endpoint connection supports CIDR overlap and is easier for network management.
> 
>   For more information, see [Set Up Private Endpoint Connections](/tidb-cloud/set-up-private-endpoint-connections.md).

Reviewer: ```suggestion
    The endpoint connection is secure and private, and does not expose your data to the public internet. In addition, the endpoint connection supports CIDR overlap and is easier for network management.

    For more information, see [Set Up Private Endpoint Connections](/tidb-cloud/set-up-private-endpoint-connections.md).
```

> **General changes**

Reviewer: ```suggestion
**Console changes**
```

> * Provide sample connection strings of MySQL, MyCLI, JDBC, Python, Go, and Node.js in the VPC Peering tab and Private Endpoint tab of the [Connect](/tidb-cloud/connect-to-tidb-cluster.md) dialog for Dedicated Tier clusters.

Author: ```suggestion
* Provide sample connection strings of MySQL, MyCLI, JDBC, Python, Go, and Node.js in the VPC Peering tab and Private Endpoint tab of the [Connect](/tidb-cloud/connect-to-tidb-cluster.md) dialog for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```

> * Provide sample connection strings of MySQL, MyCLI, JDBC, Python, Go, and Node.js in the VPC Peering tab and Private Endpoint tab of the [Connect](/tidb-cloud/connect-to-tidb-cluster.md) dialog for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.

Reviewer 2: ```suggestion
* Provide sample connection strings of MySQL, MyCLI, JDBC, Python, Go, and Node.js in the **VPC Peering** tab and **Private Endpoint** tab of the [Connect](/tidb-cloud/connect-to-tidb-cluster.md) dialog for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```

>     You can directly copy and paste the connection codes to your apps, which makes your development more productive.

Reviewer: ```suggestion
      You can easily connect to your Dedicated Tier cluster by simply copying and pasting the connection codes to your apps.
```
